### ISSUE TITLE
Get_IP_Location Failed (ssl.c:1123) SSL: Certificate Verify Failed

### ISSUE THREAD

**Issue by metalslugg3r** [tgorgdotcom/locast2plex#235]
Please help. On a Mac with Python 3.9 installed. Attempting to complete setup of Locast2Plex. Here is the error I get from terminal. I have replaced my IP Address with XXX's but the address is correct. I've tried a zipcode override but that didn't help. Thank you!

Initiating Locast2Plex v0.6.5
Opening and Verifying Configuration File.
/Volumes/Elements/locast2plex/config/config.ini
/Volumes/Elements/locast2plex/config.ini
Loading Configuration File: /Volumes/Elements/locast2plex/config.ini
Tuner count set to 3
Server is set to run on  XXXXXXXX:6077
UUID set to: xhwjrssk...
Getting user location...
Getting location via IP Address.
Error in function get_ip_location: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)
Could not acertain location.  Exiting...

**Comment by Klondike45** [comment]
Same problem here... Had this working great, then moved to a VM. Something in Windows I think is missing here...

**Comment by jpaulmaher** [comment]
I was having the same issue and found this fix online, run the following in terminal: /Applications/Python\ 3.9/Install\ Certificates.command

I received some messages about something being out of date but things are working!

**Comment by jpaulmaher** [comment]
I spoke too early.  It found the tuner, went through the guide setup, etc.  However, when I try and watch a channel, I get "Could not tune channel. Please check your tuner or antenna."

**Comment by metalslugg3r** [comment]
Yeah I get the same issue. I have to terminate and restart the script in order to get it to stream, but eventually, I get that same error and fails.

**Comment by OliRavioli** [comment]
+1
